tests/no_space_between_brackets.cpp

```diff
@@ -0,0 +1,6 @@
+{reject_missing_leading_space }
```

Comment:
Could you add_ctest() ? If we enable tests by default on a workspace that has `tools-rock_vera`, testing in the workspace will fail (because it has a test folder now). I'm usually bothered by this on third-party packages... Appreciate if we don't start doing that on our packages as well

Reply:
This is not a CMake package, so it does not apply here. This is not a full automated test suite, those are files that one can open/run vera on to check the result.

Reply:
> This is not a CMake package

It is, actually...

> This is not a full automated test 

Yeah, I was thinking something like just run vera on the files and check the output. A tiny sh script would suffice